tests/unit/test_audio_chunking.py

```
@@ -0,0 +1,275 @@
+        # File duration exactly 10 seconds, chunk duration 10 seconds
+        assert self.chunker.should_chunk(10.0) is False
+
+    def test_should_chunk_just_over_boundary(self):
+        """Test should_chunk just over boundary."""
+        # File duration 10.1 seconds, chunk duration 10 seconds
+        assert self.chunker.should_chunk(10.1) is True
+
+    @patch('audio_separator.separator.audio_chunking.AudioSegment.from_file')
+    @patch('audio_separator.separator.audio_chunking.os.path.exists')
+    @patch('audio_separator.separator.audio_chunking.os.makedirs')
+    def test_split_audio_basic(self, mock_makedirs, mock_exists, mock_from_file):
```

Ruff (0.14.11)

Unused method argument: `mock_makedirs`

(ARG002)